lms/djangoapps/courseware/module_render.py

```diff
@@ -829,10 +830,14 @@ def get_module_for_descriptor_internal(user, descriptor, student_data, course_id
     # Not that the access check needs to happen after the descriptor is bound
     # for the student, since there may be field override data for the student
     # that affects xblock visibility.
-    if getattr(user, 'known', True):
+    def _needs_access_check(user):
+        """
+        Skip access checks for unknown (LTI) users or SystemUsers.
+        """
+        return getattr(user, 'known', True) and not isinstance(user, SystemUser)
```

Comment:
I had initially planned it as a (static)method, and then realized I wasn't working in a class. :)  Local variable makes perfect sense to me.